### DISCUSSION TITLE
how to recover imei and baseband version

### DISCUSSION THREAD

**Discussion by Amit-code-eng** [community/community#55713] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i had used your opporealme ofp flash tool but i accidently emyty the blacklist .img array and then i had lost my imei and baseband version

**Comment by JesusCastroFernandez** [comment]
Recovering the IMEI and baseband version of a mobile device typically involves troubleshooting steps that may vary depending on the specific device and operating system. Here are some general methods you can try:

Check device settings: Start by checking your device's settings to see if the IMEI and baseband information is available there. On most smartphones, you can find this information under "Settings" > "About phone" or "About device." Look for sections labeled "IMEI" or "Baseband version."

Check the device's SIM tray or battery compartment: Some devices may have the IMEI number printed on the SIM tray or inside the battery compartment. Remove the SIM card or battery and look for any labels or markings that indicate the IMEI.

Dial a specific code: On certain devices, you can dial a specific code to access the IMEI information. Open the phone dialer app and enter "*#06#" (without quotes) and then press the call button. This code should display the IMEI number on the screen.

Use software tools: There are third-party software tools available for certain devices that can help retrieve IMEI and baseband information. These tools are often specific to the device manufacturer or model. You can search online for device-specific software or visit the manufacturer's official website for support tools.

Contact the device manufacturer or service provider: If none of the above methods work, it's recommended to reach out to the device manufacturer or your service provider for assistance. They may have additional methods or tools to help recover the IMEI and baseband version.

It's important to note that altering or modifying the IMEI of a device without proper authorization is illegal in many jurisdictions. Ensure that you are following the appropriate legal and ethical guidelines when attempting to recover or repair IMEI and baseband information on a device.

**Comment by mrepol742** [comment]
Most phones have their IMEI printed on the back cover. You can also find it on the your phone box and for baseband version i think you can lookit up on the internet.
